extensions/data-workspace/package.json

```diff
@@ -158,8 +160,7 @@
           "id": "dataworkspace.views.main",
           "name": "%main-view-name%",
           "contextualTitle": "%data-workspace-view-container-name%",
-          "icon": "images/data-workspace.svg",
-          "when": "isProjectProviderAvailable"
```

Comment:
I had to remove this because `isProjectProviderAvailable` gets set when the data-workspace extension gets activated